test_common/harness/deviceInfo.cpp

```diff
@@ -98,3 +98,42 @@ size_t get_max_param_size(cl_device_id device)
     }
     return ret;
 }
+
+static cl_ulong get_device_info_max_size(cl_device_id device,
+                                         cl_device_info info,
+                                         unsigned int divisor)
+{
+    cl_ulong max_size;
+
+    if (divisor == 0)
+    {
+        throw std::runtime_error("Allocation divisor should not be 0\n");
+    }
+
+    if (clGetDeviceInfo(device, info, sizeof(max_size), &max_size, NULL)
+        != CL_SUCCESS)
+    {
+        throw std::runtime_error("clGetDeviceInfo failed\n");
```

Comment:
Is this caught somewhere? It looks like a change in functionality, since the existing test_error_abort macro just returns an error from the caller and allows further testing to continue.

Reply:
I tried to harmonize this function with get_max_param_size function
Does't really make sense to continue the testing if we clGetDeviceInfo returns an error ?